### ISSUE TITLE
New page merge user flow

### ISSUE THREAD

**Issue by jsmorabito** [jsmorabito/obsidian-design#1]
https://forum.obsidian.md/t/option-to-merge-pages-from-within-theres-already-a-file-with-the-same-name-pop-up/25377

- also not accessible via keyboard only workflow
![image](https://user-images.githubusercontent.com/46250921/141662660-b8598e8f-3d22-4c72-a759-2fac054e69e3.png)
